android/src/main/java/com/instabug/reactlibrary/RNInstabug.java

```diff
@@ -0,0 +1,135 @@
+     *     It helps capture problems that occur before the React-Native bridge is established.
+     *     This offers an alternative to the {@link RNInstabugReactnativeModule#init}.
+     *     It catches start-up crashes that might slip.
+     *     This is due to a gap between the native app launch and the initialization of the react-native.
+     * </p>
+     *
+     * @param application The application context.
+     * @param applicationToken The app's identifying token, available on your dashboard.
+     * @param logLevel The level of detail in logs that you want to print.
+     *      <p>Pick one of the log levels described in {@link LogLevel}.
+     *      default logLevel is {@link LogLevel#ERROR}</p>
+     * @param instabugInvocationEvent The events that trigger the SDK's user interface.
+     *      Choose from the available events listed in {@link InstabugInvocationEvent}.
+     *
+     * @example <p>Here's an example usage: </p>
+     * <blockquote><pre>
+     * RNInstabug.initNativeSdk(
+     *     this,
+     *     "f23c....",
+     *     LogLevel.VERBOSE,
+     *     InstabugInvocationEvent.SHAKE,
+     *     ... // Other invocation events
+     * );
+     * </pre></blockquote>
+     *
+     * @example <p> you can pass an empty string in place of the last argument logLevel param and also
+     * {@link InstabugInvocationEvent#NONE} to disable all invocation events:</p>
+     *
+     * <blockquote><pre>
+     * RNInstabug.initNativeSdk(
+     *     this,
+     *     "f23c....",
+     *     new InstabugInvocationEvent[]{
+     *         InstabugInvocationEvent.NONE,
+     *     },
+     *     ""
+     * );
+     * </pre></blockquote>
+     *
+     */
+    public void init(
+            @NonNull Application application,
+            @NonNull String applicationToken,
+            int logLevel,
+            @NonNull InstabugInvocationEvent... instabugInvocationEvent
+    ) {
+        try {
+
+            setBaseUrlForDeprecationLogs();
+            setCurrentPlatform();
+
+            new Instabug.Builder(application, applicationToken, instabugInvocationEvent)
+                    .setSdkDebugLogsLevel(logLevel)
+                    .build();
+
+            // Temporarily disabling APM hot launches
+            APM.setHotAppLaunchEnabled(false);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    @VisibleForTesting
+    public void setCurrentPlatform() {
+        try {
+            Method method = InstabugUtil.getMethod(Class.forName("com.instabug.library.Instabug"), "setCurrentPlatform", int.class);
+            if (method != null) {
+                Log.i("IB-CP-Bridge", "invoking setCurrentPlatform with platform: " + Platform.RN);
+                method.invoke(null, Platform.RN);
+            } else {
+                Log.e("IB-CP-Bridge", "setCurrentPlatform was not found by reflection");
```

Comment:
We usually use `IBG` so let's match that here as well.

```suggestion
                Log.i("IBG-CP-Bridge", "invoking setCurrentPlatform with platform: " + Platform.RN);
                method.invoke(null, Platform.RN);
            } else {
                Log.e("IBG-CP-Bridge", "setCurrentPlatform was not found by reflection");
```